ui/.storybook/config.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-env node */
+import { addDecorator, addParameters, configure } from '@storybook/ember';
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
+import theme from './theme.js';
+
+addParameters({
+  viewport: { viewports: INITIAL_VIEWPORTS },
+  options: {
+    showPanel: true,
+    theme
+  },
+});
+
+addDecorator(storyFn => {
+  const { template, context } = storyFn();
+
+  const wrapperElementStyle = {
+    margin: '20px',
+  };
+
+  const applicationWrapperElement = document.createElement('div');
+  Object.assign(applicationWrapperElement.style, wrapperElementStyle);
+
+  const storybookElement = document.createElement('div');
+  storybookElement.setAttribute('id', 'storybook');
+
+  const wormhole = document.createElement('div');
+  wormhole.setAttribute('id', 'ember-basic-dropdown-wormhole');
+
+  storybookElement.appendChild(wormhole);
+
+  applicationWrapperElement.appendChild(storybookElement);
+  storybookElement.appendTo = function appendTo(el) {
+    el.appendChild(applicationWrapperElement);
+  };
+
+  return {
+    template,
+    context,
+    element: storybookElement,
+  };
+});
+
+// The order of import controls the sorting in the sidebar
```

Comment:
Hey @backspace 👋 

I had this niggle for menu link ordering for quite a while I found this recently which sorted (:badumtss:) it for me!

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#storysort-option

Seems to have changed a little bit through versions so YMMV I suppose.

Reply:
yeah, I tried using that but it was annoying because what I really wanted was to control the sorting of groups only, and have the contents of each group sort the default way. It was a hassle to specify two-level sorting like that with a function when I could instead import the directories in the order I wanted them.